scripts/ci/github_actions/driver/setup_ci_dependencies.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -xe
+set +o pipefail
+
+install_ci_dependencies (){
+  python -m pip install --upgrade pip
+  echo docker-hub==2.2.0 > dev-requirements.txt
+  pip install -r dev-requirements.txt
+}
+
+get_driver_version (){
+  source /home/runner/.bash_profile
+  cd common
+  image_version=`yq eval .identity.version config.yaml`
+  cd -
+}
+
+install_ci_dependencies
+cat >>/home/runner/.bash_profile <<'EOL'
+yq() {
+  docker run --rm -i -v "${PWD}":/workdir mikefarah/yq "$@"
+}
+EOL
+get_driver_version
```

Comment:
IMO it would be more readable if the variable assignment be here together with the `get_...` call

Reply:
I don't understand what you mean

Reply:
```suggestion
driver_version = get_driver_version()
```

Reply:
the `get_driver_version` is creating the `driver_version` variable inside him, so what is better?
1. change the variable in the function and rename the function to `create_driver_version_variable` 
2. do like you suggested and change the function will return a value

Reply:
I prefer the second option

Reply:
done